common/src/helpers/router.tsx

```diff
@@ -59,6 +59,14 @@ export const redirect = (url: string | Partial<Path>): void => {
   HISTORY.replace(url);
 };
 
+export const navigateBack = (): void => {
```

Comment:
Uh oh, you got a conflict from #1196 too.  Could you resolve this?

Reply:
Should be good to go now.